### PR TITLE
Issue 41

### PR DESCRIPTION
**Pull request by issi7915**
#### What does this PR do?
Installs dependencies upon launching the iaas_launcher.py
#### How can the reviewer verify this PR?
Run iaas_launcher.py
#### Any background context you want to provide?
Rewrote PreRequisite.sh as pre_requisites.py
#### Questions:
- Have you connected this PR to the issue it resolves?
Couldn't connect with issue-41 but associated

### CODE REVIEW

**Comment by spisarski** [comment]
Also, please see me on how to execute the CI scripts manually. I believe Randy fixed them so they should now execute.

**Comment by issi7915** [comment]
You can run the script simply using:
sudo python2 pre_requisites.py

**Comment by spisarski** [comment]
I understand how to execute it. I just do not agree that this is the correct approach.

**Comment by spisarski** [comment]
@issi7915 - I also want to ensure that you have tested this by running the CI scripts.

File: scripts/pre_requisites.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+"""
+Created on April 21, 2018
```
Comment (spisarski) [inline scripts/pre_requisites.py]: You should remove this file too

**Pull request closed without merging**